docs/content/direct/observability.md

```diff
@@ -0,0 +1,63 @@
+# Observability in KubeStellar
+
+KubeStellar provides endpoints and integrations for observability and monitoring. This page describes the available observability features, how to access them, and how to use them in a typical deployment.
+
+## Metrics Endpoints
+
+KubeStellar controllers expose Prometheus-compatible metrics endpoints. These endpoints respond to HTTP requests for metrics and can be queried by any monitoring system; KubeStellar does not mandate how metrics are collected or scraped.
+
+### Enabling Metrics
```

Comment:
The second paragraph of this section is not about enabling metrics.

The section heading "enabling metrics" is misleading, it suggests that users have to take action to enable metrics.

I suggest deleting this section heading.